content/pro/openbb-copilot.md

```diff
@@ -0,0 +1,145 @@
+---
+title: OpenBB Copilot
+sidebar_position: 8
+description: Learn how to use OpenBB Copilot to interact with the OpenBB Terminal Pro
+keywords:
+- OpenBB Copilot
+- copilot
+- voice command
+- agent
+- assistant
+- Natural language processing
+- Large language model
+- OpenAI
+---
+
+import HeadTitle from '@site/src/components/General/HeadTitle.tsx';
+import ReactPlayer from 'react-player'
+
+<HeadTitle title="Copilot | OpenBB Terminal Pro Docs" />
+
+OpenBB copilot is your companion to interact with the OpenBB Terminal Pro.
+
+## Getting started
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/open_copilot.jpg" alt="open openbb copilot" width="70%" height="70%" />
+
+To open OpenBB Copilot, click on the purple icon located at the center right of the
+Terminal Pro screen.
+
+Under most circumstances, you can treat OpenBB Copilot like your very own
+personal research assistant. OpenBB Copilot has access to the widgets on your
+dashboard, the OpenBB API and any custom endpoints you have added, as well as
+your uploaded files. Using all of these sources, OpenBB Copilot can assist you in 
+performing a wide variety of tasks.
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/copilot_opened.jpg" alt="opened openbb copilot" width="70%" height="70%" />
+
+We'll be exploring each of these features in the following sections below.
+
+## Understanding chats
+
+OpenBB Copilot is a chat-based assistant that uses the conversation history to
+help answer your queries.  This allows you to ask follow-up questions or guide
+OpenBB Copilot while perform tasks.
+
+For example, if you ask about the price-to-earnings (P/E) ratio, and then follow
+up by asking about other important financial ratios, OpenBB Copilot will
+remember the context and suggest relevant ratios:
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/pe_ratio_chat_example.jpg" alt="chat history openbb copilot" width="40%" height="40%" />
+
+If you'd like to clear the history of the current conversation, you can do so by
+clicking on the trashcan icon in the OpenBB Copilot chatbox in the upper right
+corner. It is usually a good idea to clear the chat history when you have a new
+question that is unrelated to your current conversation.
```

Comment:
@IgorWounds we'll need to add a section on new chats / deleting chats, etc. and how they work.